components/match2/commons/match.lua

```diff
@@ -243,8 +243,14 @@ function Match.copyRecords(matchRecord)
 	})
 end
 
-local function stringifyIfTable(tbl)
-	return type(tbl) == 'table' and Json.stringify(tbl) or nil
+---@param tbl string|table|nil
+---@param options {isArray: boolean?}?
+---@return string|nil
+local function stringifyIfTable(tbl, options)
```

Comment:
Is there a specific reason this isn't part of Module:Json?

Reply:
oos for this pr but yeah i plan to port it over

Reply:
nvm put it into this pr